src/index.ts

```diff
@@ -489,6 +520,13 @@ export class UnleashClient extends TinyEmitter {
             }
         }
     }
+
+    private emitReady() {
```

Comment:
its a great improvement to have this in its on little function as it is duplicated a few places! 👍🏼